responsibleai/responsibleai/_managers/counterfactual_manager.py

```diff
@@ -29,7 +29,7 @@ class CounterfactualConfig(BaseConfig):
     def __init__(self, method, continuous_features, total_CFs,
                  desired_class=CounterfactualConstants.OPPOSITE,
                  desired_range=None, permitted_range=None,
-                 features_to_vary=None):
+                 features_to_vary=None, feature_importance=False):
```

Comment:
Should we default to true?
I think PM want this.

Reply:
good point. Setting default to True.